src/load/azext_load/data_plane/load_test/custom.py

```diff
@@ -272,6 +275,33 @@ def delete_test(
     logger.info("Deleted test with test ID: %s", test_id)
 
 
+def convert_to_jmx(
+    cmd,
+    load_test_resource,
+    test_id,
+    resource_group_name=None,
+):
+    logger.info("Converting test with test ID: %s to JMX", test_id)
+    client = get_admin_data_plane_client(cmd, load_test_resource, resource_group_name)
+    existing_test = client.get_test(test_id)
+    if (
+        existing_test is not None
```

Comment:
if `existing_test` is None, need to handle this? Otherwise it will create the test?

Reply:
if there are any issues with the test itself, like it's not found, the API itself throws error, so the L286 exceptions out, don't need additional checks